src/makeRobotAccountant.js

```diff
@@ -7,6 +7,17 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      return count > 2 && count % 2 === 0
+        ? `Bzzz... Error!`
+        : a + b;
```

Comment:
Try to give more understandable names instead of `a` and `b`.

Reply:
agree